test/WakuRlnRegistry.t.sol

```diff
@@ -56,47 +58,50 @@ contract WakuRlnRegistryTest is Test {
     }
 
     function test__SingleRegistration(uint256 commitment) public {
+        vm.assume(isValidCommitment(commitment, poseidonHasher.Q()));
         wakuRlnRegistry.newStorage();
         wakuRlnRegistry.register(0, commitment);
     }
 
     function test__InvalidSingleRegistration__NoStorageContract(uint256 commitment) public {
         wakuRlnRegistry.newStorage();
+        vm.assume(isValidCommitment(commitment, poseidonHasher.Q()));
         vm.expectRevert(NoStorageContractAvailable.selector);
         wakuRlnRegistry.register(1, commitment);
     }
 
     function test__InvalidSingleRegistration__Duplicate(uint256 commitment) public {
+        vm.assume(isValidCommitment(commitment, poseidonHasher.Q()));
         wakuRlnRegistry.newStorage();
         wakuRlnRegistry.register(0, commitment);
         vm.expectRevert(DuplicateIdCommitment.selector);
         wakuRlnRegistry.register(0, commitment);
     }
 
     function test__InvalidSingleRegistration__FullTree() public {
-        vm.pauseGasMetering();
```

Comment:
Curious: any reason these gas metering calls have been removed?

Reply:
same as above, we had to pause metering because of the OutOfGas EvmError

Reply:
Ohh, that's interesting!  Thank you!